transforms/images/polus-imagej-image-integral-plugin/README.md

```diff
@@ -0,0 +1,31 @@
+# ImageJ image integral
+
+This plugin applies the image integral algorithm to an input collection.
```

Comment:
Can you add a little more description to this? Maybe add a wikipedia link to the summed area tables page?

Reply:
This has been updated.